### ISSUE TITLE
User story: Job tech primary/nice to have

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#149]
### If applicable, add a related Epic.

Epic: 

### User Story

As a recruiter, I want to categorize job technologies into 'primary' and 'nice to have', and mark some as 'learnable', to provide clearer expectations for applicants.

### Acceptance Criteria

- Technologies in job posts can be marked as 'primary', 'nice to have', or 'learnable'.
- This categorization is clear and visible to applicants.
- It helps candidates to better assess their suitability for the role.

### Please ensure you've completed these tasks before marking the issue as done.

- [ ] Reviewed the user story for clarity and completeness.
- [ ] Identified and linked related issues or epics.
- [ ] Updated the issue with any relevant notes or feedback (to date).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [ ] Develop the functionality to categorize technologies in job posts.
- [ ] Create a clear UI for this categorization in job listings.
- [ ] Ensure the system is intuitive and aids in job matching.
